be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -726,6 +732,20 @@ Status SegmentIterator::_extract_common_expr_columns(const vectorized::VExprSPtr
     return Status::OK();
 }
 
+Status SegmentIterator::_extract_common_expr_columns_for_index(const vectorized::VExprSPtr& expr) {
```

Comment:
almost the same with `_extract_common_expr_columns` could use the same function?